code/game/objects/effects/decals/Cleanable/humans.dm

```diff
@@ -20,6 +20,7 @@ var/global/list/image/splatter_cache=list()
 	var/basecolor="#A10808" // Color when wet.
 	var/list/datum/disease2/disease/virus2 = list()
 	var/amount = 5
+	appearance_flags = NO_CLIENT_COLOR
```

Comment:
There is pretty much no other way to do it, although Tiger's logic is of the type that creates maintainability issues in the first place.
To be honest part of it is that I don't think the blood sprites we use are that great in the first place, so they look pretty horrid when they're sticking out against black and white background, ESPECIALLY alien species colored blood.